clients/python/orp_at_test.py

```diff
@@ -0,0 +1,160 @@
+#============================================================================
+#
+# Filename:  orp_at_test.py
+#
+# Purpose:   Python test script to send and receive AT framed commands
+#            using the Octave Resource Protocol
+#
+# MIT License
+#
+# Copyright (c) 2020 Sierra Wireless Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+#----------------------------------------------------------------------------
+#
+# NOTES:
+#
+# 1. Install Python 3.x
+#
+# 2. Install the following, using pip3:
+#
+#    Windows:
+#    > pip3 install six pyserial
+#
+#    Linux
+#    > sudo -H pip3 install six pyserial
+#
+import logging
+import threading
+from time import sleep
+import argparse
+
+import serial
+
+import modules.orp_protocol as orp_protocol
+from modules.orp_protocol import decode_response
+from modules.orp_protocol import encode_request
+from modules.simple_at import OrpAtCmdHandler
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO)
+logger.addHandler(ch)
+
+#
+# Function to automatically reply to sync
+#
+def sync_acknowledge(data):
+    checkPacket = chr(data[0])
+
+    if orp_protocol.ORP_PKT_SYNC_SYN == checkPacket or orp_protocol.ORP_PKT_SYNC_SYNACK == checkPacket:
+        request = 'r y 0'
+        packet = encode_request(request)
+        h.sendFrame(packet.encode())
+        print('\nService Restarted\nConnected\n>')
+
+    if orp_protocol.ORP_PKT_NTFY_HANDLER_CALL == checkPacket or orp_protocol.ORP_PKT_RESP_HANDLER_CALL == checkPacket:
```

Comment:
`ORP_PKT_RESP_HANDLER_CALL` will never be sent by the Octave ORP service.  It is actually the response to `ORP_PKT_NTFY_HANDLER_CALL`.  It can be removed from this comparison - thanks.